airflow/providers/amazon/provider.yaml

```diff
@@ -548,8 +548,6 @@ extra-links:
   - airflow.providers.amazon.aws.links.logs.CloudWatchEventsLink
 
 connection-types:
-  - hook-class-name: airflow.providers.amazon.aws.hooks.s3.S3Hook
-    connection-type: s3
```

Comment:
Also remove s3 from providers connection-type, otherwise webserver (provider manager?) warn about inconsistency and in UI appear two Amazon Web Services connections.